node/cli/src/command.rs

```diff
@@ -231,6 +226,7 @@ macro_rules! construct_async_run {
 }
 
 /// Parse command line arguments into service configuration.
+#[rustfmt::skip] // This is necessary to workaround a bug in rustfmt with the function returns in the None case.
```

Comment:
Might be worth putting the returns into a separate function call if that would help remove this. It's a lot of nested code to lose formatting on.